po/gl.po

```diff
@@ -1,37 +1,75 @@
-# English translations for gcolor package.
-# Copyright (C) 2016 THE gcolor'S COPYRIGHT HOLDER
-# This file is distributed under the same license as the gcolor package.
-# Fran <frandieguez@gnome.org>, 2016.
+# Galician translations for gcolor package.
+# Copyright (C) 2013-2016 Jente Hidskes <hjdskes@gmail.com>
```

Comment:
This should be you (and the previous author, Fran Dieguez).

Reply:
As I said in #76 I think the first (C) statement is for you, and translators must be added below.